tests/utils/index.test.ts

```diff
@@ -0,0 +1,55 @@
+import { socials, formatSocial } from '../../src/utils/index';
```

Comment:
💯 Love and appreciate you did this. Will start bringing in more tests to our commands